sources/packages/backend/workflow/src/workflow-definitions/parttime-configure-disbursement.bpmn

```diff
@@ -108,7 +108,9 @@
+          <zeebe:output source="={&#10;  CSLPAmount: round half up(finalFederalAwardNetCSLPAmount / 2, 0),&#10;  CSGPAmount: round half up(finalFederalAwardNetCSGPAmount / 2, 0),&#10;  CSGDAmount: round half up(finalFederalAwardNetCSGDAmount / 2, 0),&#10;  CSPTAmount: round half up(finalFederalAwardNetCSPTAmount / 2, 0),&#10;  BCAGAmount: round half up(finalProvincialAwardNetBCAGAmount / 2, 0),&#10;  SBSDAmount: round half up(finalProvincialAwardNetSBSDAmount / 2, 0)&#10;}" target="firstHalfDisbursementValues" />
```

Comment:
Can the `firstHalfDisbursementValues` and `secondHalfdisbursementValues` use the rounding methods from the same algorithm, for instance, can we use `round half up`/`round half down` or `ceiling`/`floor`?

Reply:
`ceiling` is not recommended for rounding the first disbursement value, and `round half up` is a better option. We can certainly use `round half down` instead of `floor` to "pair with" `round half up`, but `round half down` requires two parameters -- `round down(n: number, scale: number): number` and `floor` only needs one parameter. If we use `round half down`, then `floor(floor(finalFederalAwardNetCSLPAmount) - firstHalfDisbursementValues.cslpAmount)` will be `round half down(round half down(finalFederalAwardNetCSLPAmount, 0) - firstHalfDisbursementValues.cslpAmount, 0)`. The code is a little longer, but the result is the same. I am okay either way. Please let me know if we want to change `floor` to `round half down`.

PS: `ceiling` has been replaced with `floor` in the second commit as of the comment below https://github.com/bcgov/SIMS/pull/3597#discussion_r1699128373.

Reply:
Sorry, I did not get why it would not be recommended for the "first disbursement value" and then why the `round` would be ok.
Just for clarification.

Reply:
As explained by @lewischen-aot the `ceiling` would be rounding up values between 0 and 0.5 which is not desired and that is why the `round half up` was the selected one.
The floor is also applicable for the other operation since it should act as a truncate.